components/teams-service/Makefile

```diff
@@ -40,11 +40,18 @@ proto:
 test:
 	@go test $(verbose) $(packages) -p 1 --parallel 1 -cover
 
-test_with_db:
+PG_URL ?= "postgresql://postgres@127.0.0.1:5432/teams_test?sslmode=disable&password=docker"
```

Comment:
Since you have made this file consistent, there is one more yet to do, the Makefile for authn-service, then all our auth services will have the `db_container`.

Reply:
I'll whip this up in a follow up PR so I can get rolling on data collector token fix.